src/server/middleware/twitter.middleware.ts

```diff
@@ -319,7 +317,8 @@ function loadTwitterDataV2(
   credentials: TwitterApiCredentials,
   request: GetTweetsRequest
 ): Promise<TwitterApiResponse> {
-  const requestUrl = 'https://api.twitter.com/2/tweets/search/recent';
+  const requestUrl = credentials.useEssentialOrElevatedV2 ? 'https://api.twitter.com/2/tweets/search/recent': 'https://api.twitter.com/2/tweets/search/all';
```

Comment:
Since the only thing that changes is all/recent, no need to repeat the full url
 ```suggestion
  const searchPath = credentials.useEssentialOrElevatedV2 ? 'recent' : 'all';
  const requestUrl = `https://api.twitter.com/2/tweets/search/${searchPath}`;
```